run_dir/design/deliveries.html

```diff
@@ -96,15 +96,12 @@ <h3>
         <div class="col-10 offset-1">
           <dl class="dl-horizontal hide-incoming">
```

Comment:
Don't you guys think the running note should have the same width as the FC container? Maybe the "Latest running note" could be above?

Like:

`<dl class="dl-horizontal hide-incoming">`
`<p class="mt-3" style="font-weight:bold;">Latest running note:</p>`

First note as opposed to the second:

![Skärmavbild 2023-09-26 kl  14 11 49](https://github.com/SciLifeLab/genomics-status/assets/37932118/8b32b18c-2c4e-4815-8464-ddf8209c85c8)